Gemfile

```diff
@@ -5,6 +5,8 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in feather.gemspec
 gemspec
 
+gem "minitest"
+gem "rspec-expectations"
```

Comment:
I don't see this being used.

Reply:
For some reason, I needed to add this to get the quickdraw runner to work. But I think that was when I was having issues with my installation of Ruby which I scrapped and re-setup. I just tested without these gems, and it works just fine.